backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/window/model/lookup/LabelsLookup.java

```diff
@@ -250,7 +257,8 @@ public LookupValue retrieveLookupValueById(final @NonNull LookupDataSourceContex
 	{
 		return LookupDataSourceContext.builder(tableName)
 				.setRequiredParameters(parameters)
-				.requiresAD_Language();
+				.requiresAD_Language()
+				.requiresUserRolePermissionsKey();
```

Comment:
why is `requiresUserRolePermissionsKey` needed?

Reply:
IMHO it's not needed anymore.

Reply:
true, removed.